services/frontend-v3/src/routes/Secured.js

```diff
@@ -49,16 +55,16 @@ function Secured(props) {
       });
     };
 
-    const keycloak = Keycloak("/keycloak.json");
+    const keycloak = Keycloak(keycloakJSONConfig);
     keycloak
       .init({
         onLoad: "login-required",
         promiseType: "native",
-        checkLoginIframe: false
+        checkLoginIframe: false,
       })
-      .then(authenticated => {
+      .then((authenticated) => {
         // check if user is admin
-        if (keycloak.tokenParsed.resource_access) {
+        if (keycloak.tokenParsed.resource_access["upskill-client"]) {
```

Comment:
Does this not break the code for users without roles?

Reply:
No, this is the fixed for it. I already tested it!

 `if (keycloak.tokenParsed.resource_access) {...}` --> this one break the code

`if (keycloak.tokenParsed.resource_access["upskill-client"]) {...}` --> this is the fix for it.